prow/cmd/deck/static/vendor.d.ts

```diff
@@ -10,6 +10,8 @@ interface Window {
 // Needed for mysterious reasons, otherwise ts doesn't understand this is a module
 declare module 'dialog-polyfill';
 
+declare var componentHandler: any;
```

Comment:
Can this not be of type `componentHandler` rather than `any`?

Reply:
To be honest, I don’t know the type of componentHandler, I refer to [stackoverflow](https://stackoverflow.com/questions/34800354/how-to-set-global-variable-to-material-design-lite-library-in-typescript) 

As the [JS Migration Guide](https://basarat.gitbook.io/typescript/type-system/migrating) say,  use `any`to suppressing errors?